sass/style.scss

```diff
@@ -103,5 +106,13 @@ hero {
     font-style: normal;
     font-weight: 400;
     line-height: normal;
+    margin-top: 105px;
```

Comment:
Również tutaj zamiana na rem/em. Posprawdzaj proszę czy w reszcie paddingów/marginów również nie masz px i zamień to na rem/em